var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -16,14 +16,21 @@ class Petsc(Package):
     """
 
     homepage = "http://www.mcs.anl.gov/petsc/index.html"
-    url      = "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.5.3.tar.gz"
-    git      = "https://gitlab.com/petsc/petsc.git"
-
+    git = "https://gitlab.com/petsc/petsc.git"
     maintainers = ['balay', 'barrysmith', 'jedbrown']
 
+    def url_for_version(self, version):
+        if version >= Version('3.13.0'):
+            # petsc-lite tarballs are smaller by skipping docs
+            return "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-{}.tar.gz".format(version)
+        else:
+            return "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-{}.tar.gz".format(version)
```

Comment:
I like to put this with all of the other function definitions since it doesn't change very often.

Reply:
updated